packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -87,8 +87,8 @@ module.exports = async (filename, opts) => {
    * Configure and run the transfer engine
    */
   const engineOptions = {
-    strategy: opts.conflictStrategy,
-    versionMatching: opts.schemaComparison,
+    strategy: 'restore', // for an export to file, strategy will always be 'restore'
```

Comment:
Maybe the providers should expose what strategies they support so that you cannot start a transfer from the engine with providers that don't support it, what do you think? :thinking:

Reply:
I thought about that, because I don't think I like how we've made conflictStrategy a function of the engine rather than the destination provider. I can't remember why we did that.

For now this solves the immediate problem of passing tests without breaking the CLI, and if we want to refactor it later we can.